Packs/CommonScripts/Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.py

```diff
@@ -34,34 +34,57 @@ def create_relationships_from_entity(indicator_relationships):
     return entity_relationships
 
 
-def create_indicators_loop(indicators):
+def create_indicators_loop(args, indicators):
     """ Create indicators using createNewIndicator automation
 
     :param indicators: parsed indicators
     :return: errors if exist
     """
     relationships_objects = list()
     errors = list()
+    context_data = []
```

Comment:
```suggestion
    context_data = []
    add_context = argToBoolean(args.get('add_context', False))
    tags = argToList(args.get('tags'))
```
please use `argToBoolean` and `argToList` methods